db.go

```diff
@@ -411,6 +433,35 @@ func (db *DB) Delete(list ...interface{}) (int64, error) {
 	return deleteObjects(db, db, list)
 }
 
+// Exec executes a query without returning any rows. The args are for any
+// placeholder parameters in the query.
+func (db *DB) Exec(query string, args ...interface{}) (sql.Result, error) {
+	start := time.Now()
+	result, err := db.DB.Exec(query, args...)
+	db.logQuery(stringSerializer(query), db, start, err)
+
+	return result, err
+}
+
+// Execute is a wrapper around Exec that can accept builders or strings.
+// It runs an arbitrary sequel query that does not return rows.
+func (db *DB) Execute(query interface{}, args ...interface{}) (sql.Result, error) {
```

Comment:
I think I just misunderstood the behavior when you overload a method of an embedded struct. The go `sql.DB` type already has a method named `Exec` that takes a string, so I didn't think it was possible to change the signature of `Exec`.

It makes sense now, and yeah, removing `Exec` and renaming `Execute` to `Exec` makes sense.